common/src/main/java/io/netty/util/Recycler.java

```diff
@@ -50,6 +51,7 @@ public String toString() {
     private static final int DEFAULT_MAX_CAPACITY_PER_THREAD;
     private static final int RATIO;
     private static final int DEFAULT_QUEUE_CHUNK_SIZE_PER_THREAD;
+    private static final boolean BLOCKING_POOL;
```

Comment:
We can even use a concurrent linked queue here, that has a much simpler poll operation that's not busy spinning